backend/src/main/java/hu/flowacademy/meteo/controller/StationController.java

```diff
@@ -27,4 +33,9 @@ public StationController(StationService stationService){
         log.info("Get {} (all) stationDTO.", stationDTOList.size());
         return stationDTOList;
     }
+
+    @GetMapping("hasdata")
+    public List<StationDto> listAllWithData() {
```

Comment:
Ezt service-en keresztül kellett volna hívni, nincs szükség közvetlenül a repositoryra itt. A MeasurementRepository nincs is használva?